lucene/core/src/java/org/apache/lucene/codecs/lucene90/Lucene90HnswVectorFormat.java

```diff
@@ -77,7 +77,9 @@
   static final int VERSION_CURRENT = VERSION_START;
 
   /** Sole constructor */
-  public Lucene90HnswVectorFormat() {}
+  public Lucene90HnswVectorFormat() {
+    super("Lucene90VectorFormat");
```

Comment:
historically we've used the class name as a format name, should we use
```suggestion
    super("Lucene90HnswVectorFormat");
```
?